spec/ParseLiveQuery.spec.js

```
@@ -1308,4 +1308,128 @@ describe('ParseLiveQuery', function () {
     await new Promise(resolve => setTimeout(resolve, 100));
     expect(createSpy).toHaveBeenCalledTimes(1);
   });
+
+  it('test beforeLiveQueryEvent ran while creating an object', async function () {
```

ESLint

[error] 'it' is not defined.

(no-undef)